ui/article-renderer/src/main/java/org/cru/godtools/article/activity/ArticlesActivity.java

```diff
@@ -70,6 +73,9 @@ protected void onStart() {
     public void onArticleSelected(@Nullable final Article article) {
         Timber.tag("ArticlesActivity")
                 .d("Article selected: %s", article != null ? article.title : null);
+
+        AemArticleItemActivity.start(this, Objects.requireNonNull(mTool), mLocale,
+                Objects.requireNonNull(article).uri.toString());
```

Comment:
`Objects.requreNonNull` will throw an exception if the parameter is null, this is the same as leaving the normal `NullPointerException`. what you really should be doing is `if (article != null) {}`